examples/self_contained/vowelcount/mr/reducer.py

```diff
@@ -22,5 +22,5 @@
 class Reducer(api.Reducer):
 
     def reduce(self, context):
-        s = sum(map(int, context.values))
-        context.emit(context.key, str(s))
+        s = sum(context.values)
+        context.emit(context.key, s)
```

Comment:
Not sure about the asymmetry between emitting int here and str() in base_histogram and wordcount_minimal.
It appears to me that sending either an int or a str() should have the same effect.

Reply:
Updated `wordcount_minimal` and `test_framework` with 074a45f. `base_histogram` is already emitting integers, or is there something I'm missing?